### PR TITLE
Article detail styles

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Internal link: DEVREL 886

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: lib/richTextResolvers.tsx [hunk 1]
```diff
@@ -8,35 +8,47 @@ import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/solid";
 import { Block_ContentChunk } from "../models";
 import { ReactNode } from "react";
 
-export const createDefaultResolvers = (element: Elements.RichTextElement, renderRichText: (item: Block_ContentChunk) => ReactNode): Partial<PortableTextReactComponents> => ({
+export const createDefaultResolvers = (element: Elements.RichTextElement, renderRichText: (item: Block_ContentChunk) => ReactNode, insideTable: boolean = false): Partial<PortableTextReactComponents> => ({
```
Comment (IvanKiral) [inline lib/richTextResolvers.tsx:11]: NITPICK: Maybe a little bit of tuning of insideTable variable... like `isImageInsideTable` or if more generic like `isElementInsideTable`

File: pages/articles/[slug].tsx [hunk 2]
```diff
@@ -7,27 +7,50 @@ import { GetStaticPaths, GetStaticProps } from "next";
...
+        </div>
+      </HeroImage>
+      <div className="max-w-screen-md m-auto">
+        <AuthorHorizontal item={props.article.elements.author.linkedItems[0]} />
+        <div className="flex flex-col gap-2">
+          <div className="w-fit p-2 bg-gray-800 text-white opacity-90 font-semibold">{props.article.elements.publishingDate.value && formatDate(props.article.elements.publishingDate.value)}</div>
+          <div className="flex gap-2" >
+            {
+              props.article.elements.articleType.value.length > 0 && props.article.elements.articleType.value.map(type => (
+                <div key={type.codename} className={`w-fit p-2 ${mainColorBgClass[siteCodename]} font-semibold`}>{type.name}</div>
```
Comment (IvanKiral) [inline pages/articles/[slug].tsx:44]: This is kind of weird
`mainColorBgClass[siteCodename]` does not really work.

image

However, ArticlePage has as a props siteCodename which is also used in line 28. With that one it works properly.

A quick fix is to change `siteCodename` to `props.siteCodename.`

However, it is weird that useSitecodename return wrong codename

Comment (IvanKiral) [reply pages/articles/[slug].tsx:44]: Also there is no space between `Name` and `Surname` in `authorHorizontal`

**Review by IvanKiral — CHANGES_REQUESTED** [review]
Seems good
There are only small changes requested.

The design seems good and also rich text seems to handle responsiveness good enough

File: lib/richTextResolvers.tsx (hunk 1 above)
Comment (Simply007) [reply lib/richTextResolvers.tsx:11]: `isElementInsideTable` it is

File: pages/articles/[slug].tsx (hunk 2 above)
Comment (Simply007) [reply pages/articles/[slug].tsx:44]: Fixed both - it has something to do what is rendered on server and what on client and when

Comment (Simply007) [reply pages/articles/[slug].tsx:44]: There is default value being set: `const SiteContext = createContext<CodenameContext>({ siteCodename: "ficto_healthtech" });
`

File: lib/richTextResolvers.tsx (hunk 1 above)
Comment (Simply007) [reply lib/richTextResolvers.tsx:11]: `isElementInsideTable` it is

**Review by IvanKiral — APPROVED** [review]

**Pull request merged**